rubicon/objc/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.2.3'
+__version__ = '0.2.5'
```

Comment:
I messed up. I pushed the PyPI version before checking I had all the latest Github updates.